### PR TITLE
Feature/update to support prestashop new versions

### LINKED ISSUE #22: Problemas de compatibilidad con PS 8.1.x y 8.0.x
Hola he tenido dos errores al intentar trabajar con el plug-in en las nuevas versiones.

1. Al tratar de instalar el plug-in en prestashop 8.1.x o 8.0.x obtengo errores en el guzzlehttp me da el siguiente error.

Call to undefined method GuzzleHttp\Utils::chooseHandler()

Pienso que podría ser la versión del guzzlehttp pero no tengo mucha información de cómo actualizarlo

2. Cuando voy a finalizar la venta sale error 500 trate de hacerle debug y logré saber que en el archivo redirect.php no está recibiendo la variable $cart y al enviarla null saca este error, pero no he podido saber por que no la Envia

Espero que puedan ayúdeme gracias

### PR DESCRIPTION
**Pull request by alejociro**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
   "type": "plugin",
   "require": {
-    "php": "^7.1",
+    "php": ">=7.2.5",
     "ext-curl": "*",
     "ext-json": "*",
     "ext-mbstring": "*",
-    "alejociro/redirection": "^1.0",
-    "guzzlehttp/guzzle": "~5.3"
+    "alejociro/redirection": "1.0.14",
```
Comment (freddiegar) [inline composer.json]: This is fixed value, changes this for min requirement is better

File: redirect.php [hunk 2]
```diff
@@ -19,8 +19,14 @@
         Tools::redirect('authentication.php?back=order.php');
     }
 
+    $cart = Context::getContext()->cart;
```
Comment (freddiegar) [inline redirect.php:22]: Variable to use one time? A die message looks bad in this context, it can be changes for redirect (see line 19 in this file)

**Review by freddiegar — CHANGES_REQUESTED** [review]

**Comment by freddiegar** [comment]
README file Compatibility Version section must be update to reflect changes in PHP versions and relations

File: redirect.php (hunk 2 above)
Comment (alejociro) [reply redirect.php:22]: This variable is used two times, in the validation and in the redirect method

**Review by freddiegar — APPROVED** [review]

**Pull request merged**